src/components/ha-expansion-panel.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { fireEvent } from "../common/dom/fire_event";
+import "./ha-svg-icon";
+import { mdiChevronDown } from "@mdi/js";
+
+@customElement("ha-expansion-panel")
+class HaExpansionPanel extends LitElement {
+  @property({ type: String }) title = "";
+
+  @property({ type: Boolean }) expanded = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div
+        class="summary ${this.expanded ? "expanded" : ""}"
+        @click=${this._togglePanel}
+      >
+        <div class="summary-content">
+          <span class="summary-title">
+            ${this.title}
+          </span>
+        </div>
+        <div class="summary-icon ${this.expanded ? "expanded" : ""}">
+          <ha-svg-icon path=${mdiChevronDown}></ha-svg-icon>
+        </div>
+      </div>
+      <div class="container ${this.expanded ? "expanded" : ""}">
+        <slot></slot>
+      </div>
+    `;
+  }
+
+  private _togglePanel(): void {
+    this.expanded = !this.expanded;
+    fireEvent(this, this.expanded ? "expanded" : "collapsed");
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      :host {
+        display: block;
+        width: 100%;
+        box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
+          0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+          0px 1px 3px 0px rgba(0, 0, 0, 0.12);
+      }
+
+      .summary {
+        display: flex;
+        padding: 0px 16px;
+        min-height: 48px;
+        transition: min-height 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,
+          background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
+        color: inherit;
+        border: 0;
+        cursor: pointer;
+        margin: 0;
+        outline: 0;
+        position: relative;
+        align-items: center;
+        user-select: none;
+        border-radius: 0;
+        vertical-align: middle;
+        -moz-appearance: none;
+        justify-content: center;
+        text-decoration: none;
+        background-color: transparent;
+        -webkit-appearance: none;
+        -webkit-tap-highlight-color: transparent;
```

Comment:
I think you need to go through the css and not just copy and paste from the element on md

Reply:
done, I've removed unneeded CSS, but I'm not an expert on that and I only tested on Win10+Chrome.